corpan/corpan-app/src/store/settings.ts

```diff
@@ -5,7 +5,7 @@ import { persist, createJSONStorage } from "zustand/middleware";
 import { RTL_LANGUAGES } from "./constants";
 
 export const ALL_LANGUAGES = [
```

Comment:
I think alphabetic sort, dynamic based on which language is the anchor, would be nice ... need to conquer language search soon as we break 20 languages ...